tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>can't you trigger this <code>set</code>  operation under the hood and keep the standard interface for the user?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>The setitem API does not support additional kwargs, so I cannot pass in the offset and fromrank arguments (not shown in the call in this serial notebook).  https://github.com/tskisner/pshmem/blob/master/pshmem/shmem.py#L274</p>

Reply:
<p>The notebook now shows both forms (<code>set()</code>  and explicit assignment).  For the serial case they are equivalent.  For the parallel case there is an extra communication done to find out the process where the data is coming from.</p><p> </p>